recipes/gtk-doc-stub/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+import functools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class GtkDocStubConan(ConanFile):
+    name = "gtk-doc-stub"
+    homepage = "https://gitlab.gnome.org/GNOME/gtk-doc-stub"
+    description = "Helper scripts for generating GTK documentation"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "GPL-2.0-or-later"
+    topics = ("gtk", "documentation", "gtkdocize")
+    settings = "os"
```

Comment:
Only OS ?

Reply:
it's script-only + platform independent. Hence the need for an exception in conan-hooks.
But I see that `self.settings.os` is not used at all.
So I"ll remove that.

Reply:
I was wrong :D